src/app/cases/event-trigger/case-event-trigger.component.spec.ts

```diff
@@ -30,6 +30,9 @@ class CaseEditComponent {
   @Input()
   validate: (CaseEventData) => Observable<object>;
 
+  @Input()
+  caseDetails: CaseView;
+
```

Comment:
Should we also remove unused MockComponent?
`let CaseHeaderComponent: any = MockComponent({
    selector: 'ccd-case-header',
    inputs: ['caseDetails']
  });`